src/main/java/com/targetindia/DigitizeTimeTable/repository/AdminDao.java

```diff
@@ -0,0 +1,229 @@
+        }catch (SQLException e) {
+            System.out.println(e);
+        }
+        return result;
+    }
+    public ResultSet findAllInstructors() {
+        ResultSet resultset;
+        try {
+            String query = "SELECT * FROM Instructor_table";
+            Statement statement = conn.createStatement();
+            resultset = statement.executeQuery(query);
+            return resultset;
+        } catch (SQLException e) {
+            throw new RuntimeException(String.format("No instructor available"));
+        }
+    }
+
+
+    public int deleteInstructorById(int id) {
+        try {
+            String query = "Delete from instructor_table where instructor_id = " + id;
+
+            Statement statement = conn.createStatement();
+            statement.executeUpdate(query);
+            return 1;
+        }
+        catch (Exception e){
+            return 0;
+        }
+
+    }
+
+    public void addInstructor(Instructor newInstr) {
+
+        try {
+            String query = "insert into instructor_table(instructor_id,instructor_name,contact) values(?,?,?)";
+
+            PreparedStatement statement = conn.prepareStatement(query);
+            statement.setInt(1,newInstr.getInstructor_id());
+            statement.setString(3,newInstr.getInstructor_contact());
+            statement.setString(2,newInstr.getInstructor_name());
+            statement.executeQuery();
+        }
+        catch(SQLException e){
+            System.out.println(e);
+        }
+    }
+
+    public String updateInstructor(int id, String name, long contact){
+        Statement statement;
+        try{
+            ResultSet resultset = findInstructorById(id);
+            if(resultset.next()){
+                String query="update instructor_table " +
+                        "set instructor_name='"+name+"', contact="+contact+" " +
+                        "where instructor_id="+id+" ";
+                statement=conn.createStatement();
+                statement.executeQuery(query);
+                return "Successfully updated instructor data whose id = "+ id;
+            }
+            else{
+                return null;
+            }
+
+        }
+        catch(Exception e){
+            System.out.println(e);
+        }
+        return null;
+    }
+
+    public String updateInstructor(Instructor instr){
+        PreparedStatement statement;
+        try{
+            ResultSet resultset = findInstructorById(instr.getInstructor_id());
+            if(resultset.next()){
+                String query="update instructor_table " +
+                        "set instructor_name=?, contact=? " +
+                        "where instructor_id=?";
+                statement=conn.prepareStatement(query);
+                statement.setString(1,instr.getInstructor_name());
+                statement.setString(2,instr.getInstructor_contact());
+                statement.setInt(3,instr.getInstructor_id());
+                statement.executeQuery(query);
```

Comment:
statement.executeUpdate();
```suggestion
                statement.executeQuery(query);
```